.gitattributes

```diff
@@ -1 +1,3 @@
 *.diff -text
+*.bat text eol=crlf
+*.bat text eol=lf
```

Comment:
Yes.  The first is for checkout, and the second is for checkin.  I can only assume this was a retrofit to a format which originally used one entry for specifying both.  However, while several stackoverflow posts recommend this, it's not reflected in the documentation.  So it's either undocumented, or wrong (but harmless).  Seemed to work during my Windows and Unix testing.